website/server/models/group.js

```diff
@@ -650,9 +644,8 @@ schema.methods.finishQuest = async function finishQuest (quest) {
     return await User.update({}, updates, {multi: true}).exec();
   }
 
-  let promises = [];
-  _.each(participants, userId => {
-    promises.push(_updateUserWithRetries(userId, updates));
+  let promises = _.map(participants, userId => {
```

Comment:
Nope, I just thought the standard was to use lodash functions. My mistake, I've fixed this.